src/components/views/dialogs/InviteDialog.tsx

```diff
@@ -1277,6 +1380,25 @@ export default class InviteDialog extends React.PureComponent<Props, IInviteDial
                 </AccessibleButton>
             );
 
+        let results: React.ReactNode | null = null;
+        let onlyOneThreepidNote: React.ReactNode | null = null;
+
+        if (!this.canInviteMore() || (this.hasFilterAtLeastOneEmail() && !this.canInviteThirdParty())) {
+            onlyOneThreepidNote = (
+                <div className="mx_InviteDialog_oneThreepid">
+                    {_t("Invites by email can only be sent one at a time")}
```

Comment:
This string implies the same limitation would be present when inviting into an already existing room but to my knowledge that isn't the case, this is confusing IMO

Reply:
Surrounding condition checks for `InviteKind.Dm`. Tests also cover, that it is possible to invite two emails into regular room.

Anyway I will add a comment here that we are in the DM case. :+1: